packages/extension-ui/src/Popup/Help/Restore.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ThemeProps } from '../../types';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import useTranslation from '../../hooks/useTranslation';
+import Header from '../../partials/Header';
+
+interface Props extends ThemeProps {
+  className?: string;
+}
+
+function Restore({ className }: Props): React.ReactElement<Props> {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Header
+        showBackArrow
+        showHelp
+        text={t<string>('Restore')}
+      />
+      {/* TODO: */}
+      <div className={className}>PLACEHOLDER</div>
+    </>
+  );
+}
+
+export default React.memo(
+  styled(Restore)(
+    ({ theme }: Props) => `
+  color: ${theme.textColor};
+  height: 100%;
+  overflow-y: scroll;
+  scrollbar-width: none;
+      
+  &::-webkit-scrollbar {
+    display: none;
+  }
+  `
+  )
+);
```

Comment:
I think it would be more readable if just the `div` was made a styled component, since that's the only element of `<Restore>` that those styles relate to. E.g.
```ts

const Restore = () => {
  // ...
  return (
    <>
      <Header ... />
      <Content />
    </>
  )
}

export default Restore

const Content /* or whatever name it's going to have after TODO */= styled.div`
  color: ${props => props.theme.textColor};
  height: 100%;
  overflow-y: scroll;
  scrollbar-width: none;
      
  &::-webkit-scrollbar {
    display: none;
  }
`
```

Reply:
The whole app is written in a way that they always export styled function which encapsulates component, rather  than style and return components themselves. To stay consistent with the rest of the files that we don't modify we follow the "styleguide" the extension had already and match it rather than having new components written in different way. The pattern in every component:
```js
function Table ({ children, className = '', isFull }: Props): React.ReactElement<Props> {}
// and export 
export default React.memo(styled(Table)(({ theme }: ThemeProps) => `
```
etc.

Reply:
Hmm, that doesn't sound like a convention that Styled components usage would promote 🤔 But I definitely see your point with keeping it consistent 👍 Thanks for explanation.